src/mpsc/async_impl.rs

```diff
@@ -257,6 +257,81 @@ feature! {
                 .core
                 .try_send(self.inner.slots.as_ref(), val, &self.inner.recycle)
         }
+
+        /// Returns the *total* capacity of the channel for this [`Sender`].
+        /// This includes both occupied and unoccupied entries.
+        ///
+        /// To determine the channel's remaining *unoccupied* capacity, use
+        /// [`remaining`] instead.
+        ///
+        /// # Examples
+        ///
+        /// ```
+        /// use thingbuf::mpsc::channel;
+        ///
+        /// let (tx, _) = channel::<usize>(100);
+        /// assert_eq!(tx.capacity(), 100);
+        /// ```
+        ///
+        /// Even after sending several messages, the capacity remains
+        /// the same:
+        /// ```
+        /// # use thingbuf::mpsc::channel;
+        ///
+        /// let (tx, rx) = channel::<usize>(100);
+        /// *tx.try_send_ref().unwrap() = 1;
+        /// *tx.try_send_ref().unwrap() = 2;
+        /// *tx.try_send_ref().unwrap() = 3;
+        ///
+        /// assert_eq!(tx.capacity(), 100);
+        /// ```
+        ///
+        /// [`remaining`]: Self::remaining
+        #[inline]
```

Comment:
it might be nice to add `#[must_use]` annotations to these methods, because it doesn't make sense to call them without consuming their output?
```suggestion
        #[inline]
        #[must_use]
```
(not a blocker though --- i notice that `Vec`'s `len` and `is_empty` methods don't have this in the standard library, so it may not be important...)